docs/tempo/website/configuration/_index.md

```diff
@@ -777,51 +777,56 @@ overrides:
     
     # Global ingestion limits configurations
 
-    # Specifies whether the ingestion rate limits should be applied by each instance of the distributor and ingester
-    # individually, or the limits are to be shared across all instances. See the "override strategies" section for an example.
+    # Specifies whether the ingestion rate limits should be applied by each instance
+    # of the distributor and ingester individually, or the limits are to be shared
+    # across all instances. See the "override strategies" section for an example.
     [ingestion_rate_strategy: <global|local> | default = local]
 
     # Burst size (bytes) used in ingestion.
     # Results in errors like
-    #   RATE_LIMITED: ingestion rate limit (20000000 bytes) exceeded while adding 10 bytes
+    #   RATE_LIMITED: ingestion rate limit (20000000 bytes) exceeded while
+    #   adding 10 bytes
     [ingestion_burst_size_bytes: <int> | default = 20000000 (20MB) ]
 
     # Per-user ingestion rate limit (bytes) used in ingestion. 
     # Results in errors like
-    #   RATE_LIMITED: ingestion rate limit (15000000 bytes) exceeded while adding 10 bytes
+    #   RATE_LIMITED: ingestion rate limit (15000000 bytes) exceeded while
+    #   adding 10 bytes
     [ingestion_rate_limit_bytes: <int> | default = 15000000 (15MB) ]
 
-    # Maximum size of a single trace in bytes.  `0` to disable
+    # Maximum size of a single trace in bytes.  A value of 0 disables the size
+    # check.
     # This limit is used in 3 places:
-    #  - During search traces will be skipped when they exceed this threshold.
-    #  - During ingestion traces that exceeds this threshold will be refused.
-    #  - During compaction traces that exceed this threshold will be partially dropped.
-    # During ingestion it results in errors like
+    #  - During search, traces will be skipped when they exceed this threshold.
+    #  - During ingestion, traces that exceed this threshold will be refused.
+    #  - During compaction, traces that exceed this threshold will be partially dropped.
+    # Exceeding the threshold results in errors like
```

Comment:
The language "During ingestion" was there b/c that log line will only be logged when the threshold is exceeded during ingestion.